deploy/buildspec.yml

```diff
@@ -3,6 +3,7 @@ version: 0.2
 phases:
   install:
     commands:
+      - apt-get clean -y
```

Comment:
If it doesn't solve the problem entirely. Perhaps you can explore the `--fix-broken` flag. See https://linux.die.net/man/8/apt-get